src/atoms/Heading.js

```diff
@@ -81,12 +81,12 @@ const HugeHeading = ProtoHeading.extend`
 	}
 `;
 
-const Heading = ({ size, ...rest }) => {
+const Heading = ({ size, xsize, ...rest }) => {
 	switch (size) {
 	case 'small':
 		return <SmallHeading {...rest} />;
 	case 'medium':
-		return <MediumHeading {...rest} />;
+		return <MediumHeading xsize={xsize} {...rest} />;
```

Comment:
The changes on line 84 and 89 are redundant. You can remove them without having any effect on the behavior.